Source/Core/Core/IOS/USB/Bluetooth/BTReal.h

```diff
@@ -65,10 +65,10 @@ class BluetoothReal final : public BluetoothBase
 
 private:
   static constexpr u8 INTERFACE = 0x00;
-  // Arbitrarily chosen value that allows emulated software to send commands often enough
-  // so that the sync button event is triggered at least every 200ms.
+  // Arbitrarily chosen value that allows emulated software to not block if libusb transfers
```

Comment:
Does the modification to `Java_WiimoteAdapter.java` needs to be reverted too?
I modified WiimoteAdapter java file to standardize all timeouts, but it is not really needed.